app/domain/config-types.ts

```diff
@@ -119,14 +132,27 @@ export type BarConfig = t.TypeOf<typeof BarConfig>;
 
 const ColumnFields = t.intersection([
   t.type({
-    x: GenericField,
+    x: t.type({
+      componentIri: t.string,
+      sorting: t.type({
```

Comment:
Beware! Making `sorting` non-optional makes existing chart configs invalid, thus breaking all published charts!

Reply:
I now made all sorting-related options optional. Thanks for the reminder 😅 .